cardano-crypto-tests/src/Test/Crypto/VRF.hs

```diff
@@ -106,22 +108,23 @@ testVRFAlgorithm _ n =
       ]
 
     , testGroup "verify"
-      [ testProperty "max" $ prop_vrf_max @Int @v
-      , testProperty "verify positive" $ prop_vrf_verify_pos @Int @v
+      [ {- testProperty "max" $ prop_vrf_max @Int @v
```

Comment:
Why is this property removed?

Reply:
Because 1) after the Natural -> ByteString change, it no longer compiles due to making assumptions about the seed type of a VRF being numeric, and 2) because the numeric comparison it performs is used in lieu of checking bit sizes of things.

We could test for the latter (i.e., deconstruct the seed into a bytestring, and then check whether its actual length matches the expected length); if that is an interesting property to check, then I'll happily add a test case.